### PR TITLE
Remove scm-extra

### PR DESCRIPTION
**Pull request by danepowell**
#### What it does

Remove theia/scm-extra package from browser blueprint, per advice in https://github.com/eclipse-theia/theia/issues/10786#issuecomment-1049971777

#### How to test

Verify that the (broken) History option is gone from the View menu, but no other functionality is impacted.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: applications/browser/package.json [hunk 1]
```diff
@@ -72,7 +72,6 @@
     "@theia/process": "1.40.0",
     "@theia/property-view": "1.40.0",
     "@theia/scm": "1.40.0",
-    "@theia/scm-extra": "1.40.0",
```
Comment (marcdumais-work) [inline applications/browser/package.json:75]: @danepowell could you remove the extension from the other application? It's in `applications/electron/package.json`

**Comment by marcdumais-work** [comment]
BTW, I think the CI failure is unrelated to the PR - apparently caused by a recent shortage of executor nodes for Linux.

File: applications/browser/package.json (hunk 1 above)
Comment (danepowell) [reply applications/browser/package.json:75]: Done!

**Review by marcdumais-work — APPROVED** [review]
LGTM, thanks @danepowell! I will merge after CI has passed.

**Pull request merged**